src/commcare_cloud/environment/paths.py

```diff
@@ -4,11 +4,24 @@
 import yaml
 from memoized import memoized_property, memoized
 
-REPO_BASE = os.path.expanduser('~/.commcare-cloud/repo')
-ANSIBLE_DIR = os.path.join(REPO_BASE, 'ansible')
-ENVIRONMENTS_DIR = os.environ.get('COMMCARE_CLOUD_ENVIRONMENTS', os.path.join(REPO_BASE, 'environments'))
-FAB_DIR = os.path.join(REPO_BASE, 'fab')
-FABFILE = os.path.join(REPO_BASE, 'fabfile.py')
+
+def get_virtualenv_path():
+    return os.path.dirname(sys.executable)
+
+
+def get_virtualenv_site_packages_path():
+    for filepath in sys.path:
+        if filepath.startswith(os.path.dirname(get_virtualenv_path())) and filepath.endswith('site-packages'):
+            return filepath
```

Comment:
A more conventional way to get the site-packages path is outlined here: https://stackoverflow.com/a/122340/10840
```py
from distutils.sysconfig import get_python_lib
print(get_python_lib())
```

Reply:
Oh, nice! Thanks for pointing that out. Updated